packages/debug/src/browser/debug-session.tsx

```diff
@@ -16,27 +16,27 @@
 
 // tslint:disable:no-any
 
-import * as React from 'react';
```

Comment:
Is it possible not to include re-ordering of imports?
I feel as though it is hard to see what's been changed, and it is unnecessary.

Reply:
Fixed